examples/aa-simple-dapp/src/hooks/useAlchemyProvider.ts

```diff
@@ -27,7 +34,7 @@ export const useAlchemyProvider = () => {
   );
 
   const connectProviderToAccount = useCallback(
-    (signer: SmartAccountSigner, account?: Address) => {
+    async (signer: SmartAccountSigner, account?: Address) => {
       const connectedProvider = provider
         .connect((provider) => {
           return createMultiOwnerMSCA({
```

Comment:
we should switch this over to using the new method for creating a session key signer based MSCA. It'll be good to test the default setup because that factory and executor won't use the session key if it's not installed on the account